monash-human-power/MHP_DAS
# Fixed output csvs to display a unique ID rather than the time

## Description

+ Moved the output files to the log folder
+ Files are now in the format "{ID}_{Module ID}.csv"
+ Cleaned up the pipenv files to include the new dependencies

## Screenshots

![image](https://user-images.githubusercontent.com/23159604/79221265-dc0a1b00-7e98-11ea-9e32-6734f3555789.png)
Shows the new file position and format

## Steps to Test

1. Run Raspi/Test/mqtt_wireless_log.py
2. After that has connected to MQTT brocker run Raspi/Test/mock_wireless_module.py -t 5

Hopefully the logger should record the data in the log folder!
